src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -40,10 +40,14 @@ export const WorkspaceInitial = () => {
   const isDarkTheme = uiSettings.get('theme:darkMode');
   const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
 
-  const noAdminToolTip = i18n.translate('workspace.initial.card.createWorkspace.toolTip', {
-    defaultMessage:
-      'Contact your administrator to create a workspace or to be added to an existing one.',
-  });
+  const noAdminToolTip = (
+    <>
+      {i18n.translate('workspace.initial.card.createWorkspace.toolTip', {
+        defaultMessage:
+          'Contact your administrator to create a workspace or to be added to an existing one.',
+      })}
+    </>
+  );
```

Comment:
Why was this change necessary?

Reply:
It has been solved in other pr(https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7771)